<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockedMountControlsClass</class>
 <widget class="QDockWidget" name="DockedMountControlsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mount Controls</string>
  </property>
  <widget class="QWidget" name="contents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frameMountName">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Mount: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMountName">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>434</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameControls">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0" colspan="2">
          <widget class="QFrame" name="frameButtons">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="buttonSetInitialAngles">
              <property name="text">
               <string>Set initial angles</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonRecenter">
              <property name="text">
               <string>Recenter</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupSliders">
           <property name="title">
            <string>Pan/Tilt</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSlider" name="sliderPan">
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TickPosition::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderTilt">
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="pageStep">
               <number>15</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TickPosition::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>15</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Info</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="3" column="2">
             <widget class="QLabel" name="labelTiltRange">
              <property name="text">
               <string>[0, 0]</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Tilt Angle:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLabel" name="labelPanAngle">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLabel" name="labelTiltAngle">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Pan Angle:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Pan Range: </string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="labelPanRange">
              <property name="text">
               <string>[0, 0]</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Tilt Range: </string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="buttonRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
